package.json

```diff
@@ -51,7 +51,7 @@
     "postcss-reporter": "^1.3.0",
     "postcss-resolve-nested-selector": "^0.1.1",
     "postcss-scss": "^0.1.3",
-    "postcss-selector-parser": "^2.0.0",
+    "postcss-selector-parser": "2.1.0",
```

Comment:
This was me, sorry. `2.1.1` was a patch to the parser that broke one or two of our tests. I figured I'd quickly pin it `master` and fix the tests in `v7`. `v7` is up to date with `master` and I've resolved all the conflicts yesterday.